riscv64-purecap.cmake

```diff
@@ -0,0 +1,23 @@
+set(CMAKE_SYSTEM_NAME Linux)
+
+#Set toolchain compilers
+set(CMAKE_C_COMPILER ${SDK}/bin/riscv64-unknown-freebsd13-cc)
+set(CMAKE_CXX_COMPILER ${SDK}/bin/riscv64-unknown-freebsd13-c++)
+
+
+# Don't run the linker on compiler check
+set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+#Define sysroot path for CHERI-build`
+#set(CMAKE_FIND_ROOT_PATH ${SDK}/sysroot-riscv64-purecap)
```

Comment:
In general, we don't want to include commented code (unless there's a good reason) -- I guess this is a left-over piece of debugging?